ops/services/alerts/db_metrics/main.tf

```diff
@@ -0,0 +1,25 @@
+resource "azurerm_monitor_metric_alert" "db_storage" {
```

Comment:
Are we planning to take this alert back out once Flexible Server has auto-grow implemented? If so, can we ensure we have a ticket for that on the backlog?

(Research reference, for other reviewers: https://docs.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-compute-storage#storage-auto-grow )

Reply:
We are - hopefully it's a when, not if. Captured in #3359.